aggregator/aggregator.go

```diff
@@ -223,9 +220,6 @@ func (agg *aggregator) Open() error {
 	if agg.state != aggregatorNotOpen {
 		return errAggregatorAlreadyOpenOrClosed
 	}
-	if err := agg.placementWatcher.Watch(); err != nil {
```

Comment:
We don't need to watch the placement anymore?

Reply:
I moved this call to `config/aggregator.go` since the watcher is shared between the aggregator and the flush manager.